pixel-point/pixelpoint-website
# feat: add prevent loading mobile assets

This PR prevents loading of svgs needed for mobile version on desktop
**Steps to test**
1. Open the [page](https://pixelpointwebsite-mobileassetsfix.gtsb.io/)
2. Confirm that everything looks and works as expected
